packages/api/cms-api/src/generator/utils/ts-morph-helper.ts

```diff
@@ -8,7 +8,7 @@ const project = new Project({
     tsConfigFilePath: "tsconfig.json",
     skipAddingFilesFromTsConfig: true,
 });
-function morphTsSource(metadata: EntityMetadata<any>) {
+export function morphTsSource(metadata: EntityMetadata<any>) {
```

Comment:
```suggestion
function morphTsSource(metadata: EntityMetadata<any>) {
```

This isn't needed anymore, right?

Reply:
Yes you are right, I removed the export in [ad0419b](https://github.com/vivid-planet/comet/pull/1828/commits/ad0419b5049e3697fb9869aab9c645018fe20f01)